File: backend/games/data/Game.py
from dataclasses import dataclass
from datetime import date
from typing import List


@dataclass
class Game:
    title: str
    summary: str
    platforms: List[str]
    criticsScore: int
    developers: List[str]
    publishers: List[str]
    genres: List[str]
    themes: List[str]
    releaseDate: date
    gameMinutes: int
    backgroundImage: str
    squareIcon: str
    verticalCover: str

File: src/types/game.ts
export interface game {
    title: string;
    summary: string;
    releaseKeys: string[];
    criticsScore: number | null;
    developers: string[];
    publishers: string[];
    genres: string[];
    themes: string[];
    releaseDate: Date | null;
    gameMinutes: number;
    backgroundImage: string;
    squareIcon: string;
    verticalCover: string;
}